src/bank.rs

```diff
@@ -1384,6 +1387,17 @@ impl Bank {
         self.last_ids.read().unwrap().tick_height
     }
 
+    pub fn get_storage_last_id(&self) -> Hash {
```

Comment:
@aeyakovenko pretend `self.id` is really the id of the node, is this what you're thinking of? Also, I suppose bank needs to know of the node's pubkey.

Reply:
For which problem?

Reply:
Previous PR added interface to the validator to get its state of the storage last_id, you said it should be in the account state and advertised on-chain. This has the validator create a transaction with the id it's using and then returns it from the account state when asked for it.